src/ingestion/sources/base.py

```diff
@@ -274,6 +276,10 @@ def default_for(cls, data_type):
                 cls.COMPLETENESS,
                 cls.APPROX_DISTINCTNESS,
             ]
+        if 'date' in data_type or "timestamp" in data_type:
```

Comment:
consistent quotes would be nice, where's a prettier when you need it 😭

Reply:
Fast vs. inconsistent quotes